site/content/en/docs/api_sdk/sdk/highlevel-api.md

```diff
@@ -61,3 +62,107 @@ with make_client(host="http://localhost") as client:
+resource location operations. It is the staring point for using CVAT SDK.
+
+An instance of `Client` can be created directly by calling the class constructor,
+or with the utility function `cvat_sdk.core.client.make_client()`, which can handle
+some configuration for you. The extended configuration for `Client` is performed with
+the `cvat_sdk.core.client.Config` class instances. A `Config` object can be passed to
+the `Client` constructor and then it can be accessed with the `Client.config` field.
+`Client` objects implement [context manager protocol](https://docs.python.org/3/reference/datamodel.html#context-managers).
+
+You can create `Client` this way:
+
+```python
+from cvat_sdk import make_client
+
+with make_client('localhost', port='8080', credentials=(user, password)) as client:
+    ...
+```
+
+Or, if you need to tweak `Client`, you can do this:
+
+```python
+from cvat_sdk import Config, Client
+
+config = Config()
+# set up some config fields ...
+
+with Client('localhost:8080', config=config) as client:
+    ...
+```
+
+With `Client`, you can `login()` and `logout()`, and you can get Repository objects.
+
+## Entities and Repositories
+
+Entitites represent separate objects on the server. They provide read access to object fields
+and provide additional operations, including both general Read-Update-Delete and
+object-specific ones.
+
+Repositories provide management operations for corresponding Entities. Typically, you don't
+need to create Repository objects manually. To obtain a Repository object, use the corresponding
+`Client` instance property:
+
+```python
+client.projects
+client.tasks
+client.jobs
+client.users
+...
+```
+
+An entity can be created on the server with the corresponding Repository method `create()`:
+
+```python
+task = client.tasks.create(<task config>)
+```
+
+We can retrieve server objects using the `retrieve()` method of the Repository:
+
+```python
+job = client.jobs.retrieve(<job id>)
+```
+
+After calling these functions, we obtain local objects, representing their server counterparts.
+
+Object fields can be updated with the `update()` method:
+
+```python
+job.update({'stage': 'validation'})
+```
+
+The sever object will be updated and the local object will reflect the latest object state
+after calling this operaiton.
+
+Note, that local objects may out of sync with their server counterparts for different reasons.
+If you need to update the local object with the latest server state, use the `fetch()` method:
+
+```python
+# obtain 2 local copies of the same job
+job_ref1 = client.jobs.retrieve(1)
+job_ref2 = client.jobs.retrieve(1)
+
+# update the server object with the first reference
+job_ref1.update(...)
+# job_ref2 is outdated now
+
+job_ref2.fetch()
+# job_ref2 is synced
+```
+
+The local changes in the object are discarded when `fetch()` is called.
+
+Finally, if you need to remove object from the server, you can use the `remove()` method.
```

Comment:
```suggestion
Finally, if you need to remove the object from the server, you can use the `remove()` method.
```

Also, whose method?

This should probably be supplemented with an example.

Reply:
Added an example